ramalama/model.py

```diff
@@ -188,10 +190,10 @@ def setup_container(self, args):
             conman_args += ["-e", f"{k}={v}"]
         return conman_args
 
-    def gpu_args(self, force=False, runner=False):
+    def gpu_args(self, args, runner=False):
         gpu_args = []
         if (
-            force
+            args.gpu
```

Comment:
I don't think this PR is correct, on systems with integrated graphics we don't want to use GPU acceleration, it's typically much slower than CPU inferencing, practically unusable.

With the default being a static 999 this will always be the case now, which is not good. We should look at this again before we spin up a new release.

Reply:
We can have this option but the default being 999 is not good

Reply:
I don't think I changed any existing logic here, I just added two options that allow ramalama to work on my system. The default of 999 layers is of course questionable

Reply:
It's the lack of VRAM on integrated graphics also, most of the time one will crash on one of these systems because of running out of VRAM

Reply:
Sure, but somehow ramalama works on my laptop with no gpu without any configuration. This PR did not change the default behaviour of ramalama, unless I'm missing something

Reply:
If this is try, we should try and figure out why 999 was or wasn't included on your system